lld/test/ELF/debug-names-bad-die-idx-sizes.s

```diff
@@ -1,10 +1,10 @@
-# This file was generated by first compiling main.cpp:
-# clang++ -g -O0 -S -gpubnames -fdebug-compilation-dir='debug-names-test' \
-#      main.cpp
+# This file was generated by first compiling a.cpp:
+# clang++ -g -O0 -S -gpubnames -fdebug-compilation-dir='/proc/self/cwd' a.cpp
```

Comment:
Maybe do a pass over the tests and try to have the flags similar/consistent (like this one uses `-fdebug-compilation-dir='/proc/self/cwd'`, but others use `debug-names-test` for this parameter (I'd probably err on the former rather than the later - it's uninteresting and could be consistent across all debug info tests, rather than naming it uniquely for this particular feature) - omitting `-O0`, putting the flags in the same order even/maybe, might make it easier to eyeball differences between tests.

Oh, and some tests have repro steps first, then FileCheck, some the other way around - might be nice if that was consistent. 

Maybe prose (sometimes the filename isn't enough, especially with all the DWARF, repro steps, etc, some english description of the purpose of the test might be good) description, repro steps, then RUN lines/FileCheck?

Reply:
I think you were looking at a stale version of the tests.  I have already updated them all to use '/proc/self/cwd' rather than 'debug-names-test' for the -fdebug-compilation-dir.

I will try to update the comments, etc. to be more consistent, as you've requested./

Reply:
Thanks for making all tests use `/proc/self/cwd`. Yes, it looks better.